CoEpi/ui/symptomReport/FollowUp/Cough/CoughViewController2.xib

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I noticed that the background images in some buttons (maybe all?) include the text. This is has to be changed. Text should be always added separately. Otherwise it's not possible to localize it and it may also have scaling issues. So please re-export the backgrounds without text and add the text in swift, using `L10n`. There's an article in the wiki about localization.

Also, you can use vector graphics / PDF (Figma should offer it as export format), so we don't have to manage the sizes ourselves. Importing directly PNG it's said to maybe render better results in some cases, but I haven't experienced this and it's generally not worth the hassle.